pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -1,70 +1,57 @@
-{ stdenv, fetchFromGitHub, gettext, poppler, qt5 , pkgconfig }:
+{ stdenv, fetchFromGitHub, fetchurl
+, pkgconfig, makeWrapper
+, poppler, qt5, gnuplot
+}:
 
-# Warning: You will also need a working pdflatex installation containing
-# at least auctex and pgf.
 
 # This package only builds ktikz without KDE integration because KDE4 is
 # deprecated and upstream does not (yet ?) support KDE5.
 # See historical versions of this file for building ktikz with KDE4.
 
 stdenv.mkDerivation rec {
-  version = "unstable-20161122";
-  name = "qtikz-${version}";
-
-  src = fetchFromGitHub {
-    owner = "fhackenberger";
-    repo = "ktikz";
-    rev = "be66c8b1ff7e6b791b65af65e83c4926f307cf5a";
-    sha256 = "15jx53sjlnky4yg3ry1i1c29g28v1jbbvhbz66h7a49pfxa40fj3";
-  };
+  version = "0.12";
+  pname = "qtikz";
 
   meta = with stdenv.lib; {
     description = "Editor for the TikZ language";
+    homepage = "https://github.com/fhackenberger/ktikz";
     license = licenses.gpl2;
     platforms = platforms.linux;
     maintainers = [ maintainers.layus ];
+    long_description = ''
+      You will also need a working *tex installation in your PATH, containing at least `preview` and `pgf`.
+    '';
   };
 
-  conf = ''
-    # installation prefix:
-    PREFIX = @out@
-
-    # install desktop file here (*nix only):
-    DESKTOP_INSTALL_DIR = @out@/share/applications
-
-    # install mimetype here:
-    MIME_INSTALL_DIR = @out@/share/mime/packages
-
-    # install doc here:
-    MAN_INSTALL_DIR = @out@/share/man
-
-    CONFIG -= debug
-    CONFIG += release
+  src = fetchFromGitHub {
+    owner = "fhackenberger";
+    repo = "ktikz";
+    rev = version;
+    sha256 = "1s83x8r2yi64wc6ah2iz09dj3qahy0fkxx6cfgpkavjw9x0j0582";
+  };
 
-    # qmake command:
-    QMAKECOMMAND = qmake
-    # lrelease command:
-    LRELEASECOMMAND = lrelease
-    # qcollectiongenerator command:
-    QCOLLECTIONGENERATORCOMMAND = qhelpgenerator
+  patches = [ (fetchurl {
```

Comment:
This should be `fetchpatch`. GitHub sometimes changes the diff style so we need to process it to preserve hash.

Reply:
Also please move it to a new line to make diffs in the future cleaner.
```
  patches = [
    (fetchpatch {
      …
    })
  ];
```